packages/itwin/measure-tools/src/measurements/AreaMeasurement.ts

```diff
@@ -573,6 +596,7 @@ export class AreaMeasurement extends Measurement {
 
     const jsonArea = json as AreaMeasurementProps;
     jsonArea.polygonPoints = pts;
+    jsonArea.drawingMetaData = this.drawingMetaData;
```

Comment:
Also remember to have the base json method read/write the drawing metadata.

You also have a slight problem here, the metadata has `Point2d`, but saving props it should be `XYProps`. The `Point2d` won't roundtrip correctly.

Reply:
So I have to decompose the drawingMetaData for it to process properly?

Reply:
Something like:

```
export interface DrawingMetaDataProps extends Omit<DrawingMetaData, "origin" | "extents"> {
  origin: XYProps;
  extents?: XYProps;
}
```

And the JSON methods converting it between the props-type and the "runtime-type".

Sometimes when I hit this Props problem, I'll turn the runtime-type into an actual class and not just have it as an interface, especially if there's lots of free functions that help support getting the data or doing something with it. I don't think your `DrawingMetaData` yet has a lot going on to warrant this, but just a general comment about the design philosophy.